frontend/tablet/src/components/Button.tsx

```diff
@@ -26,6 +26,7 @@ type ButtonProps = {
   title: string
   onPress: () => void
   variant?: ButtonVariant
+  style?: ViewStyle
```

Comment:
It has be also passed to the `<TouchableOpacity style={[styles.button, style]} onPress={onPress}>` to have any effect. Now it's just an unused dead optional prop.